### PR TITLE
link-to-explore-algorithms

### LINKED ISSUE #13: [Bug] <Explore Algorithms button does not redirect to tutorial page>
### Describe the bug
The Explore Algorithms button on home page does not redirect to tutorial page.

### Expected behavior
Should redirect to algorithms page or docs page

### Before clicking 
Screenshot_2024-10-02_07_31_03
### After clicking
Screenshot_2024-10-02_07_31_08

---

**Would you like to work on this issue?**  
- [x] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by Mohith1490**
### Description
I have added link to the "Explore Algorithms" button in hero section

Fixes #13 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/Homepage/HeroSection.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const HeroSection: React.FC = () => {
         {/* Call to Action Buttons */}
         <div className="flex flex-col md:flex-row items-center justify-center space-y-4 md:space-y-0 md:space-x-6">
           <a
-            href="#explore"
+            href="https://ajay-dhangar.github.io/algo/docs/"
```
Comment (ajay-dhangar) [inline src/components/Homepage/HeroSection.tsx]: Change the link in the "Explore Algorithms" button from "https://ajay-dhangar.github.io/algo/docs/" to "/docs/".

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Update your code based on the suggestions I've provided. These changes should improve the readability and maintainability of your code.

File: src/components/Homepage/HeroSection.tsx (hunk 1 above)
Comment (Mohith1490) [reply src/components/Homepage/HeroSection.tsx:21]: I think that should be /algo/docs as /algo/ is root

**Comment by Mohith1490** [comment]
Hey corrected the mistake

**Review by ajay-dhangar — APPROVED** [review]
👍

**Pull request merged**